sample/src/main/kotlin/com/slack/circuit/sample/petlist/PetList.kt

```diff
@@ -90,7 +93,7 @@ import kotlinx.parcelize.Parcelize
 data class PetListAnimal(
   val id: Long,
   val name: String,
-  val imageUrl: String,
+  val imageUrl: String?,
```

Comment:
I see why you made this nullable, but I think it's probably worth documenting when it's expected to be nullable (such as a deeplink) vs not (when coming from the pet list)

Reply:
I wasn't thinking of deeplinks here. The API returns pets with a list of image urls, which means no images would be possible. I simply updated this to reflect.

Accepting that there may not be any images makes testing easier. As the code is currently written, we expect a url and instruct Coil to load it. Accepting an empty list of images (and null here) means we use `fallback` (see https://github.com/slackhq/circuit/pull/43#discussion_r964082249) can test without having to involve the network

Reply:
What was the reasoning for make this nullable?